File: SortArrayDescending.py
# Sort Array Descending
# - Given an integer array `nums`, sort it from largest to smallest.
# - Constraints:
#   - 1 <= nums.length <= 10^4
#   - -10^4 <= nums[i] <= 10^4

from typing import List

class Solution:
    def sortDescending(self, nums: List[int]) -> List[int]:
        return sorted(nums, reverse=True)

File: SortArrayDescending.cs
// Sort Numbers in Descending Order
// - Given a list of integers nums, return the list sorted in descending order.
// - Time: O(n log n), Space: O(n)

using System;
using System.Linq;


public class Solution
{
    public int[] SortArrayDescending(int[] nums)
    {
        return nums.OrderByDescending(x => x).ToArray();
    }
}
